docs/platformcolor.md

```diff
@@ -0,0 +1,48 @@
+---
+id: platformcolor
+title: PlatformColor
+---
+
+```
+PlatformColor(string)

```

Comment:
I see in other API references that Flow notation is being used.   The full Flow notation for `PlatformColor` is: PlatformColor(...names: Array<string>): [Object object]

The PlatformColor function takes an array of strings.   The array must have at least one value.   At runtime, if the first string value does not resolve to a usable color, then the next string value is used and so on.   If none of the values resolve to a color then a RedBox error is raised.   This is useful if an app wishes to use a new platform color that is available in later OS releases but not available in earlier OS releases: the latest platform color name is specified first in the array with fallback color names specified after.

Reply:
As I re-read this, I think it makes more sense to say in English: 'The PlatformColor function takes one or more string value arguments'.   The way I wrote it above implies one passes an array literal, but one does not.   The Flow notation is somewhat confusing.   Also, it looks like I didn't properly markdown escape things.   It should be: `PlatformColor(...names: Array<string>): [Object object]`.

Reply:
I don't think it is useful to show the return type of the function. It should be completely opaque, all that matters is that it is a value that can be used as a color, anywhere a color can be used.

For reference, [`func.call`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call) uses this syntax: `func.call([thisArg[, arg1, arg2, ...argN]])` which for us would be `PlatformColor(color1, [color2, ...colorN])`